app/API/graphql_queries.js

```diff
@@ -72,3 +72,11 @@ export const loggedInUserUnreadNotificationsCount = gql`
     }
   }
 `
+
+export const loggedInUserPendingModerationCount = gql`
+  query LoggedInUserUnreadNotificationsCount {
+    loggedInUser {
+      actions_pending_moderation
```

Comment:
For Apollo's caching, it's good to always fetch the `id` (even when not using it)

```suggestion
      id
      actions_pending_moderation
```